src/qt/bitcoinamountfield.cpp

```diff
@@ -179,6 +164,25 @@ class AmountSpinBox: public QAbstractSpinBox
         return QAbstractSpinBox::event(event);
     }
 
+    StepEnabled stepEnabled() const
+    {
+        StepEnabled rv = 0;
+        if (isReadOnly()) // Disable steps when AmountSpinBox is read-only
+            return StepNone;
+        if(text().isEmpty()) // Allow step-up with empty field
```

Comment:
@jonasschnelli As I picked up a small suggestion, I also fixed your nit ^^. Can you Re-ACK this now?